samples/bluetooth/hci_rpmsg/src/main.c

```diff
@@ -33,8 +33,13 @@
 LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 /* Configuration defines */
+#if !DT_HAS_CHOSEN(zephyr_ipc_shm)
+#error "Sample requires definition of shared memory for rpmsg"
+#endif
 
-#define SHM_START_ADDR      (DT_IPC_SHM_BASE_ADDRESS + 0x400)
+#define SHM_NODE            DT_CHOSEN(zephyr_ipc_shm)
+#define SHM_BASE_ADDRESS    DT_REG_ADDR(SHM_NODE)
+#define SHM_START_ADDR      (SHM_BASE_ADDRESS + 0x400)
 #define SHM_SIZE            0x7c00
```

Comment:
@galak since we are here let's fix this as well (it should be DT_REG_SIZE() - 0x400)

Reply:
If so, please align it also here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/bluetooth/hci/rpmsg_nrf53.c#L36

Reply:
Will let that be a followup PR.